### DISCUSSION TITLE
API Server /livez and /readyz endpoints

### DISCUSSION THREAD

**Discussion by lynxz** [kubernetes-client/csharp#1084] [Q&A] [answered]
Hi is there a way to call the /livez and /readyz endpoints of the API Server it self? I'm just trying to verify the health of the cluster it self.

**Comment by tg123** [comment, marked as answer]
curl https://kubernetes.default.svc/livez

**Reply by lynxz** [reply]
Thanks for the quick reply, I was just curious if it was possible to do programmatically with the KubenetesClient, otherwise I need to build the whole http client request with the authentication myself.

**Reply by lynxz** [reply]
Which is what I did, don't spend any time with this. Thanks for the help.
